libs/hook-dapp-lib/src/utils.ts

```diff
@@ -1,6 +1,11 @@
 import * as hookDappsRegistry from './hookDappsRegistry.json'
 import { CowHook, HookDappBase } from './types'
 
+// permit() function selector
+const PERMIT_SELECTOR = '0xd505accf'
```

Comment:
@alfetopito I remember you don't like that. But we already have some orders with that values, so we have to support existing orders at least for awhile